src/argilla/training/utils.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright 2021-present, the Recognai S.L. team.
```

Comment:
These are some cool functions. I think I can use something like this for a hyperparameter tuning issue I was facing for SetFit 1.0.

Reply:
Nice🤓